tests/subsys/bootloader/bl_validation_ff_key/provision_ff.hex

```diff
@@ -0,0 +1,5 @@
+:10700000008000000040080002000000FFFFFFFFBA
```

Comment:
Same thing here about committing hex files, but where is this used?

Reply:
Thanks, it was left over from an earlier iteration. Removed